packages/devkit/devkit.1.20210517/opam

```diff
@@ -20,7 +20,7 @@ depends: [
   "ocurl" {>= "0.7.2"}
   "ocamlnet"
   "pcre"
-  "extunix" {>= "0.1.4"}
+  "extunix" {>= "0.1.4" & < "0.4.0"}
```

Comment:
this one is not needed, should be compatible and CI agrees iiuc

Reply:
my bad, reverted